examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,217 @@
+-rw-r-----  1 www-data adm       292 Apr 19 21:49 error.log.9.gz
+</pre>
+</details>
+
+In addition to log file rotation, nginx uses compression to keep the storage requirements of log files inside `/var/log/nginx/` to a minimum. After about 4 years, the `/var/log/nginx/` weighs in at just `2.7MB`:
+```bash
+du -shL /var/log/nginx/
+2.7M	/var/log/nginx/
+```
+
+The Node.js web app for Ghost on the other hand writes its logs to `/var/www/ghost/content/logs/` and that folder is already approaching `700MB` on disk: 
+```bash
+du -shL /var/www/ghost/content/logs/
+692M  /var/www/ghost/content/logs/
+```
+To keep things simple, I'll only make use of the access logs for nginx in the article.
+
+
+## Introducing a New Contender for SQL-based Observability
+[SQL-based Observability](https://clickhouse.com/blog/the-state-of-sql-based-observability) is steadily growing in popularity as an alternative to the [ELK stack](https://aws.amazon.com/what-is/elk-stack/) and the major tool at the center of this trend is ClickHouse due to its blazing-fast log-handling features. 
+
+Two metrics that help ClickHouse stand out relative to alternatives are:
+* ingestion speed and;
+* query speed.
+
+For instance, Uber's Log Analytics platform, which used to be based on ELK, could only handle up to [~25.5k docs per second](https://www.elastic.co/blog/data-ingestion-elasticsearch) compared to an ingestion speed of [up to 300K logs per second](https://www.uber.com/en-PT/blog/logging/) on a single ClickHouse node.
+
+More than 80% of their queries are aggregation queries but ELK was not designed to support fast aggregations across large datasets. This lead to very slow query speeds for aggregation queries over a 1-hour window (on a 1.3TB dataset) and frequent time outs for aggregations over a 6-hour window. 
+
+ClickHouse's columnar design allows it to support fast aggregations across large datasets out-of-the-box. The Uber team were able to further speedup the execution time of aggregation queries on ClickHouse by *materializing frequently queried fields into their own columns*. The historical analysis in the second part of this article will make use of this technique.
+
+Timeplus Proton extends the already excellent log-handling features of ClickHouse with streaming making it a perfect candidate for our first and second tasks: real-time traffic analysis and historical traffic analysis. 
+
+
+# Real-time Analysis of Web Traffic
+## Ingesting Logs using only Timeplus Proton
+The recent Timeplus article which I mentioned earlier: [Real-Time Log Stream Analysis Using an Open-Source Streaming Database](https://www.timeplus.com/post/log-stream-analysis) goes over 3 ways to stream logs to Timeplus Proton for analysis:
+
+* Filebeat + Kafka + Proton
+* Vector + Proton 
+* Proton 
+
+Out of the three options, the simplest to setup is the 3rd option where only Timeplus Proton was used for the log analysis and this is what will be used in this article.
```

Comment:
Better to put some images in the blog, even just reuse the image links in the timeplus.com/post/.. blog

Reply:
Great idea! Added.